docs/source/getting-started/quickstart-2/index.rst

```diff
@@ -208,14 +208,7 @@ Choose the best run and register it as a model. In the **Table view**, choose th
 
 Now, your model is available for deployment. You can see it in the **Models** page of the MLflow UI. Open the page for the model you just registered.
 
-You can add a description for the model, add tags, and easily navigate back to the source run that generated this model. You can also transition the model to different stages. For example, you can transition the model to **Staging** to indicate that it is ready for testing. You can transition it to **Production** to indicate that it is ready for deployment.
-
-Transition the model to **Staging** by choosing the **Stage** dropdown:
-
-.. image:: ../../_static/images/quickstart_mlops/mlflow_registry_transitions.png
-    :width: 800px
-    :align: center
-    :alt: Screenshot of MLflow tracking UI models page showing the registered model
+You can add a description for the model, add tags, and easily navigate back to the source run that generated this model. You can also apply aliases and tags to the model for easier organization and deployment. For more information, see :ref:`registry`.
```

Comment:
Ideally we would also have content on setting and using aliases here, but I wanted to keep the scope of this PR lean.